packages/web-components/src/components/callout-quote/callout-quote.scss

```diff
@@ -33,3 +33,18 @@
     margin: 1rem 0.5rem;
   }
 }
+
+:host(#{$dds-prefix}-callout-quote) {
+  .#{$prefix}--quote__wrapper {
+    margin-left: -1rem;
+    margin-right: -1rem;
+    @include carbon--breakpoint('md') {
+      margin-left: -1rem;
+      margin-right: -1rem;
```

Comment:
```suggestion
    margin-left: -$spacing-05;
    margin-right: -$spacing-05;
    @include carbon--breakpoint('md') {
      margin-left: -$spacing-05;
      margin-right: -$spacing-05;
```

Reply:
Thanks @emyarod .
I've updated the style with spacing token.